src/tests/ftest/cart/ctl/cart_ctl_five_node.py

```diff
@@ -79,10 +76,10 @@ def test_cart_ctl(self):
 
         time.sleep(5)
 
-        clicmd = self.utils.build_cmd(self, self.env, "test_clients_1")
-        self.utils.launch_test(self, clicmd, srv_rtn)
-        clicmd = self.utils.build_cmd(self, self.env, "test_clients_2")
-        self.utils.launch_test(self, clicmd, srv_rtn)
+        for i in range(2):
+            clicmd = self.utils.build_cmd(self, self.env,
+                                          "test_clients", index=i)
```

Comment:
PEP8 standards [recommend](https://www.python.org/dev/peps/pep-0008/#names-to-avoid) not using `i` .  Also you can break long lines on [method arguments](https://www.python.org/dev/peps/pep-0008/#indentation) to give you more room, e.g.
```suggestion
        for index in range(2):
            clicmd = self.utils.build_cmd(
                self, self.env, "test_clients", index=index)
```